lib/stitches/generator_files/app/controllers/api/api_controller.rb

```diff
@@ -1,10 +1,20 @@
 class Api::ApiController < ActionController::Base
   include Stitches::Deprecation
+  #
```

Comment:
I can remove this massive comment if people find it annoying. I thought the ordering was interesting and unintuitive, which is why I added it in.